styx-api-service/src/main/java/com/spotify/styx/api/SchedulerProxyResource.java

```diff
@@ -29,6 +29,9 @@
 import com.spotify.apollo.Response;
 import com.spotify.apollo.route.AsyncHandler;
 import com.spotify.apollo.route.Route;
+
+import java.net.MalformedURLException;
```

Comment:
```suggestion
```

Sorry I missed this as well. I think you can configure IntelliJ to do import optimization upon saving. :)